tests/tfa_basic8.test

```diff
@@ -0,0 +1,375 @@
+<?php
+
+/**
+ * @file tfa_basic.test.
+ * Drupal test cases for TFA basic plugins.
+ */
+
+/**
+ * Tests the functionality of the TFA basic plugins.
+ */
+class TfaBasicTestCase extends DrupalWebTestCase {
+
+  /**
+   * @var PHPGangsta_GoogleAuthenticator
```

Comment:
I think this library has been deprecated and is also used in the GA_Login module.

Is there a reason why it's showing up in TFA basic?